tests/components/vizio/test_media_player.py

```diff
@@ -495,9 +507,15 @@ async def test_setup_with_apps_exclude(
     caplog: pytest.fixture,
 ) -> None:
     """Test device setup with apps and apps["exclude"] in config."""
-    await _test_setup_with_apps(
-        hass, MOCK_TV_WITH_EXCLUDE_CONFIG, CURRENT_APP, CURRENT_APP_CONFIG
-    )
+    async with _cm_for_test_setup_tv_with_apps(
+        hass, MOCK_TV_WITH_EXCLUDE_CONFIG, CURRENT_APP_CONFIG
+    ):
+        attr = hass.states.get(ENTITY_ID).attributes
+        _assert_source_list_with_apps(list(INPUT_LIST_WITH_APPS + [CURRENT_APP]), attr)
+        assert CURRENT_APP in attr["source_list"]
+        assert attr["source"] == CURRENT_APP
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5373293)